_data/projects/couchers.yml

```
@@ -0,0 +1,13 @@
+name: Couchers
+desc: Couchers.org is an online platform for the global couch-surfing community.
+site: https://couchers.org/
+tags:
+  - python
+  - sqlalchemy
+  - grpc
+  - javascript
+  - reactjs
+  - typescript
+upforgrabs:
+  name: good first issue
+  link: https://github.com/Couchers-org/couchers/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

```suggestion
  link: https://github.com/Couchers-org/couchers/labels/good%20first%20issue
```
That would be better.